agent/command/git_merge_pr.go

```diff
@@ -71,24 +71,23 @@ func (c *gitMergePr) Execute(ctx context.Context, comm client.Communicator, logg
 		return errors.Wrap(err, "unable to get patch")
 	}
 
-	_, projectToken, err := getProjectMethodAndToken(c.Token, conf.Expansions.Get(evergreen.GlobalGitHubTokenExpansion), conf.Distro.CloneMethod)
-	if err != nil {
-		return errors.Wrap(err, "failed to get token")
+	token := c.Token
+	if token == "" {
+		token = conf.Expansions.Get(evergreen.GlobalGitHubTokenExpansion)
```

Comment:
getProjectMethodAndToken also split the expansion, to use just the token and not the word "token". Is this something we definitely don't need to consider anymore?